treeherder/perf/migrations/0040_performancealert_noise_profile.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.1.12 on 2021-07-16 14:14
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('perf', '0039_store_more_job_details_on_record'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='performancealert',
+            name='noise_profile',
+            field=models.CharField(
+                choices=[
+                    ('SKEWED', 'Samples are heavily found on one side of the mean.'),
+                    (
+                        'OUTLIERS',
+                        'There are more outliers than should be expected from a normal distribution.',
+                    ),
+                    (
+                        'MODAL',
+                        'There are multiple areas where most values are found rather than only one.',
+                    ),
+                    ('OK', 'No issues were found.'),
+                    ('N/A', 'Could not compute a noise profile.'),
+                ],
+                default='N/A',
+                help_text='The noise profile of the data which precedes this alert.',
+                max_length=30,
+            ),
+        ),
+    ]
```

Comment:
Has the django migration/schema change been tested on prototype? If its a very large table, it might be a good idea to do so before a prod deploy.